continuous_integration/scripts/build-cunumeric-cpp

```diff
@@ -9,7 +9,7 @@ build_cunumeric_cpp() {
     cmake_args+=(-DBUILD_MARCH=${BUILD_MARCH:-haswell});
     cmake_args+=(-DCMAKE_BUILD_TYPE=Release);
     cmake_args+=(-DCMAKE_CUDA_ARCHITECTURES=RAPIDS);
-    cmake_args+=(-Dlegate_core_ROOT=$HOME/legate/build);
+    # cmake_args+=(-Dlegate_core_ROOT=$HOME/legate/build);
```

Comment:
Since this is only used with the associated CI scripts, can we just change the path instead of commenting it out?